pkg/virtctl/guestfs/guestfs.go

```diff
@@ -41,6 +41,7 @@ const (
 	applianceDir      = "/usr/local/lib/guestfs"
 	guestfsVolume     = "guestfs"
 	appliancePath     = applianceDir + "/appliance"
+	guestfsHome       = "/home/guestfs"
```

Comment:
will this be fine for guestfs pods that were requested with `--root`?

Reply:
I think so because the emptyDir is created with the permission for uid used by the pod at runtime. We also have a functional test that tries to run with this option https://github.com/kubevirt/kubevirt/blob/main/tests/storage/guestfs.go#L214

Reply:
I can add a statement that set the path differently if you prefer

Reply:
right we have a test, no that is fine